tests/_compilertestrunner.livecodescript

```diff
@@ -0,0 +1,637 @@
+﻿script "CompilerTestRunner"
+/*
+Copyright (C) 2015 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+-- FIXME provide this on the command line
+constant kLogFilename = "_test_suite.log"
```

Comment:
Probably need to call this "_compiler_test_suite.log" to avoid a conflict with the LCS tests.  Or fix both of them to accept a `--logfile` argument.